examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -279,6 +279,7 @@ export default class App extends PureComponent {
       }
     });
 
+    tileset.setOptions({loadTiles: true});
```

Comment:
This is the default setting for a new Tileset3D, right? Do we need to explicitly set it?

I probably don't understand what problem you are fixing.

Reply:
The problem is that if a customer signed the checkbox "Load tiles" to false and goes to some other tile it stays false but by default its's true and it should loads tiles

Tam gave the task to us to fix this.
Probably it's not the best solution but I'll figure out what's the actually problem of this tiles behavior

Reply:
> I probably don't understand what problem you are fixing.

so full refactoring of debug panel is here https://github.com/visgl/loaders.gl/pull/1403

The architecture of debug panel is more transparent and responsive now :)